pkgs/development/python-modules/deemix/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, spotipy
+, click
+, pycryptodomex
+, mutagen
+, requests
+, deezer-py
+}:
+buildPythonPackage rec {
+  pname = "deemix";
+  version = "3.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-FGzMFJOoKQPNq4tGI1KsMO+i9iBZhoz5Z67BFLEuv48=";
+  };
+
+  propagatedBuildInputs = [
+    spotipy
+    click
+    pycryptodomex
+    mutagen
+    requests
+    deezer-py
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://git.freezer.life/RemixDev/deemix-py";
+    description = "deemix is a deezer downloader built from the ashes of Deezloader Remix.";
```

```suggestion
    description = "Deezer downloader built from the ashes of Deezloader Remix";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md